examples/visual-tests/amp-consent/amp-consent.js

```diff
@@ -15,25 +15,29 @@
  */
 'use strict';
 
-const {verifySelectorsVisible} = require('../../../build-system/tasks/visual-diff/helpers');
+const {
+  verifySelectorsVisible,
+} = require('../../../build-system/tasks/visual-diff/helpers');
 
+// TODO(wg-analytics, #25105) flaky test.
 module.exports = {
+  'flaky': true,
```

Comment:
You can revert this file and add a single line here:

https://github.com/ampproject/amphtml/blob/0e05b6da02211e9d943f23137009214cbffc1e55/test/visual-diff/visual-tests#L170-L174

Reply:
done